internal/attestation/gcp/validator_test.go

```diff
@@ -164,7 +164,7 @@ Y+t5OxL3kL15VzY1Ob0d5cMCAwEAAQ==
 	}
 }
 
-func mustMarshal(in any, require *require.Assertions) []byte {
+func mustMarshal(in *attest.GCEInstanceInfo, require *require.Assertions) []byte {
```

Comment:
The GCP Issuer doesn't marshal a pointer, but the object itself, so this function should probably do the same

Reply:
If I do this, we get a different warning: 
```
mustMarshal passes lock by value: github.com/google/go-tpm-tools/proto/attest.GCEInstanceInfo contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex
```

the `state protoimpl.MessageState` field holds a `pragma.DoNotCopy`.

Reply:
In that case we should update the gcp validator and issuer to also work on a pointer to `attest.GCEInstanceInfo`.

Reply:
Updated all usages in this package